### PR TITLE
Airport attempt

### PR DESCRIPTION
**Pull request by breadmaking**

### CODE REVIEW

File: lib/airport.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    @planes = []
+  end
+
+  def landing(plane)
+    raise 'No free spots to land - Aiport Full' if @planes.count == @capacity
+    @planes.push(plane)
+  end
+
+  def take_off(plane)
+    @planes.remove(plane)
```
Comment (heykathl) [inline lib/airport.rb:14]: Did you mean @planes.pop?

File: spec/airport_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require './lib/airport'
+
+describe Airport do
+  it 'responds to take off' do
```
Comment (heykathl) [inline spec/airport_spec.rb:4]: Adding in a describe '#takeoff' above line 4 to group them into a context block

File: spec/airport_spec.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    airport = Airport.new(20)
+    expect(airport).to respond_to(:take_off)
```
Comment (heykathl) [inline spec/airport_spec.rb:6]: When I call airport.take_off(plane), it seems that the test isn't expecting an argument. Perhaps expect(airport).to respond_to(:take_off).with(1).argument?

**Review by lykathleen — COMMENTED** [review]
Code was clear and easy to understand - coverage is 96.15%. 
README was not updated. As the challenge was not completed, there was no explanation on where and how the challenge would be completed. 
A suggestion is to run rubocop which improved the formatting of code.

**Pull request closed without merging**